src/conditions/BUILD

```diff
@@ -30,6 +30,15 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "linux_mips64el",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:mips64el",
```

Comment:
Is this defined?
I wonder about the need for mips64el specificially as the first case.
I'm not up on my mips naming. Is the el designation different from other mips64 instruction sets?

Reply:
Only `mips64` is defined at https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD#L119

@zqzzq Can you double check this?